lib/src/native/realm_bindings.dart

```diff
@@ -11293,15 +11293,16 @@ abstract class realm_errno {
   static const int RLM_ERR_SYNC_CLIENT_RESET_REQUIRED = 1032;
   static const int RLM_ERR_SYNC_COMPENSATING_WRITE = 1033;
   static const int RLM_ERR_SYNC_CONNECT_FAILED = 1034;
-  static const int RLM_ERR_SYNC_INVALID_SCHEMA_CHANGE = 1035;
-  static const int RLM_ERR_SYNC_PERMISSION_DENIED = 1036;
-  static const int RLM_ERR_SYNC_PROTOCOL_INVARIANT_FAILED = 1037;
-  static const int RLM_ERR_SYNC_PROTOCOL_NEGOTIATION_FAILED = 1038;
-  static const int RLM_ERR_SYNC_SERVER_PERMISSIONS_CHANGED = 1039;
-  static const int RLM_ERR_SYNC_USER_MISMATCH = 1040;
-  static const int RLM_ERR_TLS_HANDSHAKE_FAILED = 1041;
-  static const int RLM_ERR_WRONG_SYNC_TYPE = 1042;
-  static const int RLM_ERR_SYNC_WRITE_NOT_ALLOWED = 1043;
+  static const int RLM_ERR_SYNC_CONNECT_TIMEOUT = 1035;
```

Comment:
I think RLM_ERR_SYNC_CONNECT_TIMEOUT is a new value and it should be added to the Dart enum also.